test/test_snopt_workspace.py

```diff
@@ -0,0 +1,122 @@
+"""
+This example is taken from the OpenOpt Examples website.
```

Comment:
Is this the same as the large_sparse but with dense Jacobians? Can you generalize that file instead of making a new file with (mostly) duplicate code for the setup portion?

Reply:
Yes, I'll do this.